ansible/roles/vi/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+  - name: Create vim directory
+    file:
+      path=/home/vagrant/{{ item.dir }}
+      state=directory
+      owner=vagrant
+      group=vagrant
+      mode=0775
+    with_items:
+    - { dir: '.vim' }
+
+  - name: Creates directory
+    file:
+      path=/home/vagrant/.vim/{{ item.dir }}
```

Comment:
can you reformat to match the `:` separated key values, like in the other roles please. just basing this on the examples given on the ansible roles docs